test/Frontend/unknown-arguments.swift

```diff
@@ -6,3 +6,9 @@
 // RUN: not %swiftc_driver -c %s -o %t.o -Xfrontend -fake-frontend-arg -Xfrontend fakevalue 2>&1 | %FileCheck -check-prefix=XFRONTEND %s
 
 // XFRONTEND: <unknown>:0: error: unknown argument: '-fake-frontend-arg'
+
+// RUN: not %swift -D@#%! -DSwift=Cool -D-D -c %s -o %t.o 2>&1 | %FileCheck -check-prefix=CMDLNCOND %s
```

Comment:
Can you add a test (or even just an argument) that shows the split `-D`, as mentioned above?

(Also, nitpick: there's no need to pick a compact name for the FileCheck prefix. "INVALID-COND" or something is fine.)